DataFormats/TrackReco/interface/TrackBase.h

```diff
@@ -138,7 +138,8 @@ namespace reco {
       hiRegitMuTobTecStep = 43,
       hiRegitMuMuonSeededStepInOut = 44,
       hiRegitMuMuonSeededStepOutIn = 45,
-      algoSize = 46
+      siStripTripletStep = 46,
+      algoSize = 47
```

Comment:
How about using the value of `reservedForUpgrades1` instead? (at the time I meant it exactly for this kind of situation)

Reply:
makes sense; I guess the idea is to rename it.
BTW, isn't the `siStripTriplet` confusing? pixel-less iteration is also a strip triplet iteration
also, what if down the road we decide to make it use 4 hits?

Reply:
> makes sense; I guess the idea is to rename it.

Exactly, do
```diff
       detachedQuadStep = 24,
-      reservedForUpgrades1 = 25,
+      siStripTiripletStep = 25,
       reservedForUpgrades2 = 26,
```
That would avoid the class version changes.

Reply:
it might be that a new dedicated step (again targeting displaced tracking) would be added for Run3, in that case how should we proceed ?

Reply:
From what I see there is room for another one (reservedForUpgrades2)
Replacing reservedForUpgrades1 by SiStripTripletStep indeed makes it much simpler without changing much the structure (already tried)

Reply:
if another "regional" displaced iteration is planned, how about we  plan to use the `reservedForUpgrades1` and `reservedForUpgrades2`
and name the proposed iteration `displacedGeneralStep` while the one from Andrew will be `displacedRegionalStep`?

Reply:
thanks, sounds good to me, indeed